samples/servicecontrol/retry-messages/sample.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Retry failed messages using ServicePulse
+summary: A sample showing how to retry failed messages
+component: Core
+reviewed: 2024-09-03
+---
+
+This sample shows how to retry a failed message. The sample uses the [Learning Transport](/transports/learning/) and a portable version of the Particular Service Platform tools. Installing ServiceControl is **not** required.
+
+include: platformlauncher-windows-required
```

Comment:
I must admit I copied it from the repair message sample. I guess the include did not age well.

Reply:
I'm wondering if instead of adding the include we could add a note like this:

> This sample includes a portable version of the Particular Service Platform tools. These tools are available only in Windows environments.